bin/docker-save.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
This runs in CI to save docker images as tar archives so they can be included in individual packages.